internal/integration/helpers.go

```diff
@@ -319,3 +362,24 @@ func Eventually(tb testing.TB, times uint64, f func() error) {
 func IndexFilePath() string {
 	return path.Join(FileNameRoot, "index.txt")
 }
+
+func newSigningKey(tb testing.TB) *signingKey {
```

Comment:
Does it make sense to move this to the utils package too?

Reply:
Moved it